src/main/java/com/syncstream_dev/syncstream/model/message/Message.java

```diff
@@ -0,0 +1,25 @@
+package com.syncstream_dev.syncstream.model.message;
+
+public abstract class Message {
+    protected String mid;     // Message id
```

Comment:
Avoid abbreviations like `mid`.

Reply:
a747393f2eed0e5af27d61d62611a5176b43979a addresses this